client/src/components/GasInfoHead.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function GasInfoHead({ gasInfoHead, fuelValue }) {
+  return (
+    <>
+      {gasInfoHead === fuelValue.charAt(0).toUpperCase(0) + fuelValue.slice(1) ? (
+        <Wrapper>{gasInfoHead}</Wrapper>
+      ) : (
+        <></>
+      )}
+    </>
+  );
+}
+
+const Wrapper = styled.h2`
+  position: fixed;
+  padding: 10px;
```

Comment:
If you zoom your app, the font-size gets bigger, but the padding remains the same - this might looks strange at the end.

Reply:
Thank you for the advice.

Reply:
Ups :D Thanks 👍